hack/dev/openstack-ceph.yaml

```diff
@@ -0,0 +1,114 @@
+apiVersion: core.openstack.org/v1beta1
```

Comment:
only negative about having an example like this within the cinder-operator is it is likely to become stale quickly. Would this specific example be better suited within the openstack-operator itself?

Reply:
My idea of having this example here for now was to promote cinder folks *always* deploy using the openstack-operator and for now to make sure we *always* keep it updated.

In the near future (once we merge the `extraVolumes` stuff) I'm think of adding it like you suggest to the openstack-operator itself and make the `openstack_deploy` target in `install_yamls`' Makefile to do the ceph _toy_ cluster deployment automatically and use it in Cinder and Glance, that way we get something similar to what devstack does, a working OpenStack deployment with Ceph.

Reply:
Maybe we can merge PR77 with `openstack-ceph.yaml` and then remove it once a copy of it is in the openstack-operator.